[DM-4903] new smartrest template 532 (#3716)
SoftwareAG/c8y-docs: content/smartrest/mqtt-static-templates.md

<li>[528,serial,softwareToBeUpdated1,version1,url1,action1,sw2,ver2,url2,action2,...](#528)</li>
<li>[529,serial,softwareToBeUpdated1,version1,type1,url1,action1,sw2,ver2,type2,url2,action2,...](#529)</li>
<li>[530,serial,hostname,port,connectionKey](#530)</li>

> <li>[532,serial,PARAM_DEF_0,PARAM_DEF_1,...,PARAM_DEF_N](#532)</li>

Reviewer: ```suggestion
<li>[532,serial,parameterKey1,type1,value1,...](#532)</li>
```

```

##### Parameter update operation (532) {#532}

> Notifies device that **c8y_ParameterUpdate** operation has been created. 

Reviewer 2: ```suggestion
Notifies a device that a **c8y_ParameterUpdate** operation has been created. 
```

> Notifies a device that a **c8y_ParameterUpdate** operation has been created. 

Reviewer 2: ```suggestion
Notifies the device that a **c8y_ParameterUpdate** operation has been created. 
```

| Position | Parameter          | Type   | Description                                                         |
|:---------|:-------------------|:-------|:--------------------------------------------------------------------|
| 1..n     | list of parameters | n/a    |                                                                     |
| 1.1      | parameter name     | string | i.e. MyParameter                                                    |
| 1.2      | data type          | string | "s" - string, "n" - number, "b" - boolean, "" - (empty string) null |

> | 1.3      | value              | string | value serialized to string or empty string for null values          |

Reviewer: ```suggestion
| 1...     | list of 3-sets per parameter | n/a    |                                                                     |
| 1.1      | parameter name     | string | i.e. MyParameter                                                    |
| 1.2      | data type          | string | "s" - string, "n" - number, "b" - boolean, "" - (empty string) null |
| 1.3      | value              | string | value serialized to string or empty string for null values          |
```

**Example**

Operation with fragments:

```json
{
  "c8y_ParameterUpdate": {},
  "c8y_ParameterUpdate_MyParameter": {},
  "c8y_ParameterUpdate_DeviceMaintainer": {},
  "c8y_ParameterUpdate_Problems": {},
  "MyParameter": true,
  "DeviceMaintainer": {
    "name": "John Smithsky",
    "contact": "12312"
  },
  "Problems": null
}
```

> Will result in message:

Reviewer 2: ```suggestion
Will result in the message:
```

```text
532,someSerial,MyParameter,b,true,DeviceMaintainer.name,s,John Smithsky,DeviceMaintainer.contact,s,12312,Problems,,
```

> Maximum number of parameters is **100** - operations containing more parameters won't be sent to device.

Reviewer 2: ```suggestion
The maximum number of parameters is **100**. Operations containing more parameters won't be sent to the device.
```

> The maximum number of parameters is **100**. Operations containing more parameters won't be sent to the device.

Reviewer: ```suggestion
The maximum number of parameters is **100**. Operations containing more parameters cannot be pushed through this template.
```
